app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -217,17 +218,6 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
         'tax_class_id',
     ];
 
-    /**
-     * Links attribute name-to-link type ID.
-     *
-     * @var array
-     */
-    protected $_linkNameToId = [
```

Comment:
Please keep this property for backward compatibility purpose, it's better to mark it deprecated instead of deletion

Reply:
okay, so basically that will be the default if not di config is passed? tricky, but should work ..